src/loaders/svg/tvgSvgSceneBuilder.cpp

```diff
@@ -284,8 +284,12 @@ static void _applyComposition(SvgLoaderData& loaderData, Paint* paint, const Svg
 
             bool isMaskWhite = true;
             if (auto comp = _sceneBuildHelper(loaderData, compNode, vBox, svgPath, true, 0, &isMaskWhite)) {
-                Matrix finalTransform = _compositionTransform(paint, node, compNode, SvgNodeType::Mask);
-                comp->transform(finalTransform);
+                if (!compNode->node.mask.userSpace) {
```

Comment:
probably this condition should be moved to 233 line?

```
if (!compNode->node.clip.userSpace && !compNode->node.mask.userSpace) {
  ....
}
```

Reply:
I have already tried that, but then only the first icon is displayed correctly (you ca find the svg file in the issue → Test image Godot icons :

![fix_mtx](https://github.com/thorvg/thorvg/assets/4047289/0bd1f6e5-5782-403d-b490-9e7940416e11)

Reply:
Never mind, this seems to be `windows` issue...

Reply:
@capnm you mean my comment is valid?